src/ray/object_manager/object_manager.cc

```diff
@@ -609,6 +609,8 @@ bool ObjectManager::ReceiveObjectChunk(const NodeID &node_id,
     // have to check again here because the pull manager runs in a different
     // thread and the object may have been deactivated right before creating
     // the chunk.
+    RAY_LOG(INFO) << "Aborting object creation because it is no longer actively pulled: "
```

Comment:
Should this be `INFO` or `DEBUG`?

Reply:
I think this one is okay as INFO since it should not happen as often (object fetch races with the local memory capacity dropping) and is useful to report.

Reply:
Sounds good.